docs/guides/monitor/dimension-templates.md

```diff
@@ -1,6 +1,6 @@
 <!--
 title: "Use dimension templates to create dynamic alarms"
-custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/health/dimension-templates.md
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/monitor/dimension-templates.md
```

Comment:
> /monitor/monitor/

Reply:
[btw there is no `dimension-templates.md`](https://github.com/netdata/netdata/tree/master/docs/monitor)